internal/controller/nimcache_controller.go

```diff
@@ -463,7 +464,7 @@ func (r *NIMCacheReconciler) reconcileJobStatus(ctx context.Context, nimCache *a
 			return fmt.Errorf("failed to get selected profiles: %w", err)
 		}
 
-		if len(selectedProfiles) > 0 {
+		if len(selectedProfiles) > 0 && !utils.ContainsElement(selectedProfiles, "all") {
```

Comment:
Add a const string definition for "all"?

Reply:
sure